test/function/priority.cpp

```diff
@@ -35,6 +35,20 @@ namespace {
     EXPECT_EQ( result, 119 );
   }
 
+  TEST( GetPriorityValue, NumTooHighPriority ) {
+    int result;
+
+    result = stumpless_prival_from_string( "192" );
+    EXPECT_EQ( result, -1 );
```

Comment:
Also call `EXPECT_ERROR_ID_EQ` after these and make sure that the error is set to the appropriate type.

Reply:
done